tests/test_chargebee_all_fields.py

```diff
@@ -0,0 +1,316 @@
+from tap_tester import connections, runner, menagerie
+from base import ChargebeeBaseTest
+
+class ChargebeeAllFieldsTest(ChargebeeBaseTest):
+
+    # fields that are common between V1 and V2
```

Comment:
A mismatch between field name and comment, do provide detailed comment if you are skipping these fields.

Reply:
Updated the comment.